__tests__/app.test.js

```diff
@@ -1,7 +1,12 @@
 const endpointsJson = require("../endpoints.json");
-/* Set up your test imports here */
+const request = require("supertest");
+const db = require("../db/connection");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
 
-/* Set up your beforeEach & afterAll functions here */
+beforeAll(() => seed(data));
```

Comment:
beforeAll will seed the database before all tests run. What happens if you post or delete data in any one of the tests? How can you account for the change in data between each test? Hint - you deleted this comment in the code _/* Set up your beforeEach & afterAll functions here */_